plugins/modules/api.py

```diff
@@ -451,7 +451,6 @@ def ros_api_connect(self, username, password, host, port, use_ssl):
                     conn_status["connection"]["port"] = port
                 ctx = ssl.create_default_context()
                 ctx.check_hostname = False
-                ctx.set_ciphers('ADH:@SECLEVEL=0')
```

Comment:
@NikolayDachev was there a reason why you added this? Does it not work for you without this line? If yes, we should add a setting for it.

(We should also do that for the line above, `check_hostname`.)

Reply:
I did some testing, with this line I could neither connect to my device when "TLS version" was set to "any", nor when it was set to "only v1.2". So if there's no explanation why this line is needed on some devices, let's just remove it. We can make it configurable if someone reports that it is really needed in some cases.